File: mobile/src/main/java/com/thewizrd/simplewear/ScreenLockAdminReceiver.java
package com.thewizrd.simplewear;

import android.app.admin.DeviceAdminReceiver;
import android.content.Context;
import android.content.Intent;

import androidx.annotation.NonNull;

/**
 * Receiver class which shows notifications when the Device Administrator status
 * of the application changes.
 */
public class ScreenLockAdminReceiver extends DeviceAdminReceiver {
    @Override
    public void onEnabled(@NonNull Context context, @NonNull Intent intent) {
    }

    @Override
    public void onDisabled(@NonNull Context context, @NonNull Intent intent) {
    }
}

File: mobile/src/main/java/com/thewizrd/simplewear/ScreenLockAdminReceiver.kt
package com.thewizrd.simplewear

import android.app.admin.DeviceAdminReceiver
import android.content.Context
import android.content.Intent

/**
 * Receiver class which shows notifications when the Device Administrator status
 * of the application changes.
 */
class ScreenLockAdminReceiver : DeviceAdminReceiver() {
    override fun onEnabled(context: Context, intent: Intent) {}
    override fun onDisabled(context: Context, intent: Intent) {}
}